diff --git a/setup.py b/setup.py
index 14aa80f..c4fa2f4 100644
--- a/setup.py
+++ b/setup.py
@@ -1,8 +1,7 @@
 import os
 import warnings
-from packaging import version
 import versioneer
-import pip
+from setuptools._vendor.packaging import version
 from setuptools import setup
 from setuptools import Command
 from setuptools.command.test import test
@@ -12,23 +11,6 @@ from Cython.Distutils.build_ext import new_build_ext as build_ext
 from Cython.Build import cythonize
 import numpy
 
-if version.parse(pip.__version__) < version.parse("19.0"):
-    # version 19.0 is required, since we are using pyproject.toml for definition of build-time depdendencies. See
-    # https://pip.pypa.io/en/stable/news/#id209
-    warnings.warn(
-        "You are using pip version {}. However, pip version > 19.0 is recommended. You can continue with the "
-        "installation, but installation problems can occour. Please refer to "
-        "https://github.com/precice/python-bindings#build-time-dependencies-cython-numpy-defined-in-pyprojecttoml-are"
-        "-not-installed-automatically for help.".format(
-            pip.__version__))
-
-if version.parse(pip.__version__) < version.parse("10.0.1"):
-    warnings.warn(
-        "You are using pip version {}. However, pip version > 10.0.1 is required. If you continue with installation "
-        "it is likely that you will face an error. See "
-        "https://github.com/precice/python-bindings#version-of-pip3-is-too-old".format(
-            pip.__version__))
-
 # name of Interfacing API
 APPNAME = "pyprecice"
 
